terraform/install.sh

```diff
@@ -75,22 +75,35 @@ getBillingAccount() {
 }
 
 getProject() {
-  log "Checking for project list"
+  log "Checking for project list..."
+  acct=$(gcloud info --format="value(config.account)")
+  # get projects with user account
+  owner_projects=$(gcloud projects list --filter="id:stackdriver-sandbox-* AND name='Stackdriver Sandbox Demo'" --format="value(projectId)")
```

Comment:
Where is the filter for owned projects? I see you filter the projects that have Stackdriver Sandbox Demo in their name

Reply:
I see that you make the determination of owner later on. The name owner_projects here is then misleading.